specs/start_screen/01_main.rst

```diff
@@ -0,0 +1,30 @@
++-------+--------------+-----------+------------+
+| issue | title        | author    | created    |
++-------+--------------+-----------+------------+
+| 86_   | Start screen | tschubotz | 2019-01-17 |
++-------+--------------+-----------+------------+
+
+.. _86: https://github.com/gnosis/safe/issues/86
+
+Start screen
+============
+
+1. `Main`_
+2. `User Interface`_
+
+.. _Main:
+
+.. contents:: Table of Contents
+    :depth: 3
+
+1. Problem Definition
+---------------------
+
+- We have update our Safe messaging and slogan a while ago
+  ("Enter the decentralized web").
+- We need to update that inside the app
+
+Inputs, outputs and use case scenarios are omitted since they are trivial.
+Please check `User Interface`_ for the screens.
```

Comment:
Shouldn't this incluse the use cases for the start password button (or at least mention them or link to another epic)?

Reply:
yes and no. It depends on how we split up the tickets. This ticket was only about updating the slogan. That's why I left it out.

Reply:
Ah, you mean we should mention that this button starts the flow. okay, will add a note.